src/pathfinding_service/service.py

```diff
@@ -231,20 +230,21 @@ def handle_channel_closed(self, event: ReceiveChannelClosedEvent) -> None:
         log.info("Received ChannelClosed event", event_=event)
 
         token_network.handle_channel_closed_event(channel_identifier=event.channel_identifier)
-        self.database.delete_channel_views(event.channel_identifier)
+        self.database.delete_channel(event.token_network_address, event.channel_identifier)
 
     def handle_message(self, message: Message) -> None:
+        changed_channel: Optional[Channel]
```

Comment:
Should you set a default of `None` here or is this implicit?

Reply:
There is no default. It *will* get initialized or we won't reach a place where it's used. I only needed to add the type declaration because otherwise mypy would assume `Channel` instead of `Optional[Channel]`.

Would you prefer a different way to write it?

Reply:
Ah, I missed the `return` there. I'd still feel that `changed_channel: Optional[Channel] = None` is more explicit.